cmd/iso8583/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+var (
+	programName = filepath.Base(os.Args[0])
+	describeCmd = "describe"
+)
+
+func main() {
+	describeCommand := flag.NewFlagSet("describe", flag.ExitOnError)
+	flag.Usage = func() {
+		fmt.Fprintf(os.Stdout, "Work seamlessly with ISO 8583 from the command line.\n\nUsage:\n  %s <command> [flags]\n\n", programName)
+		fmt.Fprintf(os.Stdout, "Available commands:\n")
+
+		// TODO: we will print all commands when we have more than one
+		fmt.Fprintf(os.Stdout, "  %s: display ISO 8583 file in a human-readable format\n", describeCmd)
+		fmt.Fprintf(os.Stdout, "\n")
+	}
+
+	describeCommand.Usage = func() {
+		fmt.Fprintf(os.Stdout, "Display ISO 8583 file in a human-readable format.\n\nUsage:\n  %s %s <files> [flags]\n\n", programName, describeCmd)
```

Comment:
This is a nitpick, but I've found Go's backtick strings are handy here. No need to change - just mentioning. 

```suggestion
		fmt.Fprintf(os.Stdout, `Display ISO 8583 file in a human-readable format.
		
Usage:
  %s %s <files> [flags]
`, programName, describeCmd)
```